eventstore/store_usage_events_test.go

```diff
@@ -1,1295 +0,0 @@
-package eventstore_test
-
-import (
-	"encoding/json"
-	"time"
-
-	"github.com/alphagov/paas-billing/eventio"
-	"github.com/alphagov/paas-billing/eventstore"
-	"github.com/alphagov/paas-billing/testenv"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("GetUsageEvents", func() {
-
-	var (
-		cfg eventstore.Config
-	)
-
-	BeforeEach(func() {
-		cfg = testenv.BasicConfig
-	})
-
```

Comment:
Look at tests for config -> eventstore we have tests for db insert and requests but not code level stuff currently